client/src/components/DateInput.jsx

```diff
@@ -0,0 +1,42 @@
+import styled from 'styled-components';
+
+const DateInput = ({
+  title,
+  text_content,
+  required,
+  min,
+  max,
+  disabled,
+  value,
+  myFunction,
+}) => {
+  return (
+    <Wrapper>
+      <label htmlFor={title}>{text_content}</label>
+      <Input
+        type="date"
+        id={title}
+        name={title}
+        min={min}
+        max={max}
+        required={required}
+        disabled={disabled}
+        value={value}
+        onInput={myFunction}
+      ></Input>
+    </Wrapper>
+  );
+};
+export default DateInput;
+
+const Wrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+`;
+
+const Input = styled.input`
+  height: 3rem;
+  padding: 0.5rem;
+  background-color: white !important;
```

Comment:
Why do you need `!important`here? Usually, if `!important` is used that indicates that it is not exactly known why the declaration alone is not working. If my assumption is true you should check why you need to set `!important` and think about a better solution.

Reply:
@mbosselmann Thank you for the hint. !important was obviously quick and dirty. :-) I found out now how to do it properly with webkit-calendar-picker-indicator.